plat/rpi3/include/platform_def.h

```diff
@@ -181,6 +181,14 @@
 #define BL32_DRAM_BASE			SEC_DRAM0_BASE
 #define BL32_DRAM_LIMIT			(SEC_DRAM0_BASE + SEC_DRAM0_SIZE)
 
+#if LOAD_IMAGE_V2
+#ifdef SPD_opteed
+/* Load pageable part of OP-TEE at end of allocated DRAM space for BL32 */
+#define RPI3_OPTEE_PAGEABLE_LOAD_SIZE	0x080000 /* 512KB */
+#define RPI3_OPTEE_PAGEABLE_LOAD_BASE	(BL32_DRAM_LIMIT - RPI3_OPTEE_PAGEABLE_LOAD_SIZE) /* 0x1028_0000 */
```

Comment:
Actually, I would prefer it if you removed the address in the comment in line 188. This could become outdated if BL32 is moved to a different address.